### PR TITLE
{vis}[GCCcore/10.2.0] zbar 0.23.91 

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/z/zbar/zbar-0.23.1-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+homepage = 'https://github.com/mchehab/zbar'
+description = """ZBar is an open source software suite for reading bar codes
+from various sources, including webcams. As its development stopped in 2012,
+I took the task of keeping it updated with the V4L2 API. This is the main
+repository for it. There's a clone at at LinuxTV.org, and another one at gitlab."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/mariusmuja/flann/archive/']
```
Comment (zao) [inline easybuild/easyconfigs/z/zbar/zbar-0.23.1-GCCcore-10.2.0.eb:18]: This source URL seems to be incorrect, copy-paste error?

Comment (sassy-crick) [reply easybuild/easyconfigs/z/zbar/zbar-0.23.1-GCCcore-10.2.0.eb]: Erm, yes, copy&paste error. Thanks for spotting that.

**Comment by sassy-crick** [comment]
@zao In light quite some time has passed, how shall we proceed here?

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/z/zbar/zbar-0.23.91-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    ('Java', '11.0.2', '', SYSTEM),
+    ('Qt5', '5.14.2'),
+]
+
+configure_cmd_prefix = "autoreconf -vfi && "
+#
+# uncomment if you want don't want to build the documents
+# configopts = "--disable-doc --with-gtk=auto --with-python=auto"
+osdependencies = ['xmlto']
+configopts = " --with-gtk=auto --with-python=auto"
```
[the remark this answers is not in the document]
Comment (sassy-crick) [reply easybuild/easyconfigs/z/zbar/zbar-0.23.91-GCCcore-10.2.0.eb]: Sounds good to me. I have changed that. Thanks

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 5 out of 7 (2 easyconfigs in total)
himem05.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/jfgrimm/ff118a105d2a81bbef431eefd3053944 for a full test report.

**Comment by sassy-crick** [comment]
@jfgrimm 
Do you have any idea why it tries to do this:
```
libtool: install: /usr/bin/install -c python/.libs/zbar.soT /home/sassy/apps/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/zbar.so
/usr/bin/install: cannot create regular file '/home/sassy/apps/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/zbar.so': Permission denied
```
Is that due to a change in the `Python` version between `Gcc-10.2.0` and `Gcc-11.2.0` ?

In my opinion it should not try and install anything in `Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages` at all. 

Odd that it worked on Generoso (btw, I did not see you already submitted the test job)

**Comment by boegel** [comment]
closing this since `GCC/10.2.0` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**